ui/tests/integration/components/pki/page/pki-configuration-details-test.js

```diff
@@ -182,74 +168,11 @@ module('Integration | Component | Page::PkiConfigurationDetails', function (hook
     this.version = this.owner.lookup('service:version');
     this.version.version = '1.13.1';
     await render(
-      hbs`<Page::PkiConfigurationDetails @urls={{this.urls}} @crl={{this.crl}} @mountConfig={{this.mountConfig}} @hasConfig={{true}} />,`,
+      hbs`<Page::PkiConfigurationDetails @urls={{this.urls}} @crl={{this.crl}} @hasConfig={{true}} />,`,
       { owner: this.engine }
     );
     assert.dom(SELECTORS.rowValue('Cross-cluster revocation')).doesNotExist();
     assert.dom(SELECTORS.rowValue('Unified CRL')).doesNotExist();
     assert.dom(SELECTORS.rowValue('Unified CRL on existing paths')).doesNotExist();
   });
-
```

Comment:
Removing these test since the mount config now lives in the route template. There are also component tests for the `SecretsEngineMountConfig` component!